xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/attachment_macros.vm

```diff
@@ -55,94 +55,98 @@
   #set ($attachments = $filteredAttachments)
 #end
 
-#macro (displayAttachments $attachments)
```

Comment:
I would still be nice to have a re-usable Velocity macro to display the attachments of a given document, based on the live table.